### PR TITLE
Revise a comment in EvalIJOffsetSmooth()

### PR DESCRIPTION
**Pull request by xuechen417**
- Revise the comment and related variable naming to reflect the code behavior correclty

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 83e0cde9418e5057e05f2208fefde85e4766136a
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>PAL: dev 40af910391fb8c287cb37bf520c41310bf88d405</li>
<li>SPVGEN: dev ce06cb5e3116ba77a22c3278dfeadfd865a8977c</li>
<li>XGL: dev 7e13a8bd0bb57d3cfb3bc014f6b26a8c9bb8bfd9</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 08268e9955d48ca075b239ae46328694ddff2413</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 939/939 (100.0%)</li>
<li>Failed: 0/939 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 65433/65433 (100.0%)</li>
<li>Failed: 0/65433 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 65465/65465 (100.0%)</li>
<li>Failed: 0/65465 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Navi10</summary><ul>

<li>Passed: 65465/65465 (100.0%)</li>
<li>Failed: 0/65465 (0.0%)</li>
</ul></details>


</ul></details>

File: builder/llpcBuilderImplInOut.cpp [hunk 1]
```diff
@@ -552,13 +552,13 @@ Value* BuilderImplInOut::EvalIJOffsetSmooth(
     Value* pPullModel = ReadBuiltIn(false, BuiltInInterpPullMode, {}, nullptr, nullptr, "");
     // Adjust each coefficient by offset.
     Value* pAdjusted = AdjustIJ(pPullModel, pOffset);
-    // Extract <I,J> part of that, and divide by W.
-    Value* pIJ = CreateShuffleVector(pAdjusted, pAdjusted, { 0, 1 });
+    // Extract <I/W, J/W, 1/W> part of that
+    Value* pIJDivByW = CreateShuffleVector(pAdjusted, pAdjusted, { 0, 1 });
```
Comment (amdrexu) [inline builder/llpcBuilderImplInOut.cpp]: pIJDivByW -> pIJDivW

**Comment by JaxLinAMD** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit d00df5533877b86c9118c117718ad1e03dd0c3ff
<details open><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master b601c88aeca7a7b08becb3d32709de383c8ee428</li>
<li>METROHASH: amd-master 2b6fee002db6cc92345b02aeee963ebaaf4c0e2f</li>
<li>PAL: dev 40af910391fb8c287cb37bf520c41310bf88d405</li>
<li>SPVGEN: dev ce06cb5e3116ba77a22c3278dfeadfd865a8977c</li>
<li>XGL: dev 7e13a8bd0bb57d3cfb3bc014f6b26a8c9bb8bfd9</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 08268e9955d48ca075b239ae46328694ddff2413</li>
</ul></details>

<details open><summary>ShaderDB test</summary><ul>

<li>Passed: 939/939 (100.0%)</li>
<li>Failed: 0/939 (0.0%)</li>
</ul></details>

<details open><summary>CTS tests (built with branch Vulkan-CTS-1.1.2.3-cherry-picks)</summary><ul>


<details open><summary>Ubuntu 18.04, Ellesmere</summary><ul>

<li>Passed: 65433/65433 (100.0%)</li>
<li>Failed: 0/65433 (0.0%)</li>
</ul></details>

<details open><summary>Rhel 7.6, Vega10</summary><ul>

<li>Passed: 65465/65465 (100.0%)</li>
<li>Failed: 0/65465 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Navi10</summary><ul>

<li>Passed: 65465/65465 (100.0%)</li>
<li>Failed: 0/65465 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**